end-to-end-tests/src/test/java/tech/pegasys/peeps/consensus/GoQuorumCliqueConsensusTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.peeps.consensus;
+
+import tech.pegasys.peeps.NetworkTest;
+import tech.pegasys.peeps.NodeConfiguration;
+import tech.pegasys.peeps.SignerConfiguration;
+import tech.pegasys.peeps.network.ConsensusMechanism;
+import tech.pegasys.peeps.network.Network;
+import tech.pegasys.peeps.node.Account;
+import tech.pegasys.peeps.node.Web3ProviderType;
+import tech.pegasys.peeps.node.model.Hash;
+import tech.pegasys.peeps.node.verification.ValueReceived;
+import tech.pegasys.peeps.node.verification.ValueSent;
+
+import org.apache.tuweni.eth.Address;
+import org.apache.tuweni.units.ethereum.Wei;
+import org.junit.jupiter.api.Test;
+
+public class GoQuorumCliqueConsensusTest extends NetworkTest {
```

Comment:
Would be possible to parameterise this test for Besu or GoQuorum. This and the CliqueConsensusTest almost exactly the same except the line 37.

Reply:
yes we should, but I _think_ for now, we get such bang-for-our-buck just having GoQuorum in, that I don't really want to mess around with niceties - i.e. I'm _personally_ happy with this tech debt ... but I can be convinced if really unhappy :)